core/drivers/clk/fixed_clk.c

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2021, Bootlin
+ */
+
+#include <assert.h>
+#include <drivers/clk.h>
+#include <drivers/clk_dt.h>
+#include <initcall.h>
+#include <io.h>
```

Comment:
initcall.h & io.h not needed

Reply:
Also removed assert, kernel/* and assert.